### ISSUE TITLE
Can't see my thermostat in list of devices

### ISSUE THREAD

**Issue by mdegat01** [allenporter/python-google-nest-sdm#5]
I'm a user of Home Assistant which leverages this library for its [nest integration](https://www.home-assistant.io/integrations/nest/). I noticed today that my thermostat disappeared so I dug in to investigate.

I installed this library directly and set up an offline oauth client and SDM project for it to use. I was able to successfully go through the authentication steps. During those steps I was prompted to allow access to my thermostat and did:
<img width="747" alt="Screen Shot 2021-01-30 at 4 16 32 PM" src="https://user-images.githubusercontent.com/2037026/106368296-937ff900-6316-11eb-9578-c8d5ea9be38c.png">

But then afterwards when I ran `google_nest --project_id="${PROJECT_ID}" list_devices` and my thermostat didn't show up, it only found my 2 cameras and my doorbell. Here's what I got (filtered out the irrelevant details out with jq to keep output small):
<img width="988" alt="Screen Shot 2021-01-30 at 4 28 32 PM" src="https://user-images.githubusercontent.com/2037026/106368542-39803300-6318-11eb-8733-225b5c9fcf64.png">

Not really sure what I could be doing wrong at this point since it clearly did authenticate and find my other devices, seems like there's an issue? Not sure what capabilities the library provides for debugging but let me know and I'll run them and/or collect logs.

**Comment by allenporter** [comment]
My impression is that if you sent raw commands to the API you'll be missing the thermostat, e.g. following the instructions in https://developers.google.com/nest/device-access/use-the-api -- if you want to confirm that, it would narrow down that it is not a bug in the python library, and instead is an issue with the google side not returning the thermostat.

I hear about this happening fairly often, and added some troubleshooting steps in https://www.home-assistant.io/integrations/nest/#troubleshooting

What worked for me was restarting the thermostat (see link in the section that starts with "Thermostat not created" above).    (I've only ever heard about this happening with thermostats)

Let me know how it goes and happy to help more.

**Comment by mdegat01** [comment]
You're right, that did the trick. After restarting its back in the list.

I guess I'll keep an eye on it but I'm a bit concerned. I mean the fix is obviously easy enough once recognized but I set up the integration in HA 2 days ago. So 2 days ago my my thermostat was there and today it disappeared until I restarted it. I mean I realize that's a low sample size so maybe its a one-off. But if I have to restart my thermostat every couple days to keep it online that's going to be a problem, its a pretty key part of my home automation.

Is there an open bug somewhere for the SDM API about this I can add a comment to and follow?
